workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectManager.cs

```diff
@@ -8,21 +8,32 @@ public class GameObjectManager
     {
         private readonly Dictionary<Entity, GameObject> entityGameObjectMapping = new Dictionary<Entity, GameObject>();
 
+        public bool HasGameObjectEntity(Entity entity)
+        {
+            return entityGameObjectMapping.ContainsKey(entity);
+        }
+
         public void AddGameObjectEntity(Entity entity, GameObject gameObject)
```

Comment:
And as a driveby, could this be `AddGameObjectToEntity` ?

Reply:
Obsolete now as the method was removed.